lib/vmpooler/providers/base.rb

```diff
@@ -120,6 +124,14 @@ def migrate_vm_to_host(_pool_name, _vm_name, _dest_host_name)
         end
 
         # inputs
+        #   [String] pool_name      : Name of the pool
+        #   [String] vm_name        : Name of the VM to migrate
+        #   [Class] redis           : Redis object
+        def migrate_vm(_pool_name, _vm_name, _redis)
```

Comment:
Apart from Redis, how is this method signature different from `migrate_vm_to_host`?.  Is it just simply calling

`migrate_vm_to_host(pool_name, vm_name, nil)` ?

Redis is already a global variable, so no ed to pass it through.  That should probably be a separate PR

Reply:
The main difference to me, beyond the redis part which as you point out shouldn't be there, is that there's no longer a need to provide the destination host name, so I figured keeping migrate_vm instead of migrate_vm_to_host made sense. I can change it if it makes more sense.